packages/tracing-internal/src/browser/metrics/index.ts

```diff
@@ -201,14 +201,56 @@ function _trackFID(): () => void {
   });
 }
 
+enum InteractionType {
+  Click = 'click',
+  Hover = 'hover',
+  Drag = 'drag',
+  Press = 'press',
```

Comment:
Let's avoid using enums, they suck for bundle size. See https://blog.sentry.io/performance-impact-of-generated-javascript/#switching-from-typescript-enums-to-const-and-string-enums

Reply:
Updated to just plain strings!